ipfs-api/src/lib.rs

```diff
@@ -188,13 +188,11 @@ extern crate derive_more;
 #[macro_use]
```

Comment:
_drive by PR commented here, don't mind me_

You might want to consider removing all these extern crate declarations (they're not required in the 2018 edition)

Reply:
I know that, thanks.
Just fixed enough for migration purposes.